Formula/iperf.rb

```diff
@@ -16,6 +16,11 @@ class Iperf < Formula
   def install
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
+
+    # Otherwise this definition confuses system headers on 10.13
+    # https://github.com/Homebrew/homebrew-core/issues/14418#issuecomment-324082915
+    inreplace "config.h", "#define bool int", ""
```

Comment:
I think it will also start failing on 10.12 when newer Xcode is used. And it definitely doesn't hurt to undefine it everywhere.